docs/conf.py

```diff
@@ -10,13 +10,15 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
+import sys
+sys.path.append("..")
 import pdbx
 
 # -- Project information -----------------------------------------------------
 
 project = "mmcif_pdbx"
 author = "John Westbrook"
-release = pdbx.__version__
+version = pdbx.__version__
```

Comment:
IIRC, `release` and `version` are two different things for the sphinx templates. I _think_ one was just MAJOR.MINOR the other the full MAJOR.MINOR.PATH-WHATEVER/ You'll have to check if both are needed (and if the distinction makes sense for you). A safe bet is
```suggestion
version = release = pdbx.__version__
```

Did you build the docs and do they look ok?